support-data-rest/src/test/resources/init.sql

```diff
@@ -44,7 +44,8 @@ ALTER TABLE uniprotkb_statistics_entry
 CREATE TABLE attribute_query
 (
     id   SERIAL PRIMARY KEY,
-    attribute_name  varchar(64),
+    attribute_name  varchar(64) NOT NULL,
+    statistics_category_id int8 NOT NULL,
     query varchar(64)
 );
```

Comment:
Minor comment - To mimick the actual table, we can also add unique key constraint on attribute_name and statistics_category_id in H2